### ISSUE TITLE
example_run_landmarks_recognition.py hangs

### ISSUE THREAD

**Issue by itsderek23** [luxonis/depthai-python-extras#32]
When running `python3 example_run_landmarks_recognition.py` , the video stream hangs immediately after displaying the first frame. I'm seeing this on the RPi Compute and we have a report of the same issue from another user (unknown board).

[See the log](https://gist.github.com/itsderek23/75006ae53cd6e29fff74242508f89054).

**Comment by itsderek23** [comment]
On another run, I saw a stream of the errors below. I'm not seeing this on every run.

```
Device get data failed: 7
E: [         0] [python3] XLinkReadDataWithTimeOut:1468	/home/pi/depthai/mdk/common/components/XLink/shared/XLink.c:1468
	 Assertion Failed: link != NULL
```

**Comment by itsderek23** [comment]
Fixed by 98fb7753650fbc0326fdff31a10df758f98dab9e.

**Comment by itsderek23** [comment]
Re-opened due to c152ab74ab6e65344b0fb0adba87a528cabc76ad
